### ISSUE TITLE
Change activation status by clicking on the icon

### ISSUE THREAD

**Issue by nedroden** [blekerfeld/donut#66]
It would be helpful if you could change the activation status of a language by clicking its status icon in the list of languages.

**Comment by blekerfeld** [comment]
Good idea! Let's do this by adding an extra route, somehow combining forces of pDatafield and pAction.
